src/Interpreters/Aggregator.cpp

```diff
@@ -3100,6 +3100,9 @@ void Aggregator::mergeBlocks(BucketToBlocks bucket_to_blocks, AggregatedDataVari
                 if (bucket > max_bucket)
                     break;
 
+                if (!bucket_to_blocks.contains(bucket))
```

Comment:
2 questions:
* A general one about the function: Shouldn't `bucket_to_blocks` be a reference?
* And about this change: Why? It seems the for-loop after checking is_cancelled is going to do the exact same as this if.

Reply:
* it is a reference (we capture a reference to it)
* just seemed logical to me. not needed for correctness, indeed

Reply:
> it is a reference (we capture a reference to it)

It's not in this function. There are 2 `Aggregator::mergeBlocks` and this one does not use a reference and passes a copy